gapic/src/main/com/google/gapid/perfetto/views/StyleConstants.java

```diff
@@ -70,10 +70,8 @@
 
     public final RGBA textMain;
     public final RGBA textAlt;
-    public final RGBA textInvertedMain;
```

Comment:
The code change is alright, but I still have to say I will miss the white text color on slices, I like it (QAQ).